src/vuex/modules/extension.js

```diff
@@ -22,10 +22,12 @@ export default () => {
   const actions = {
     async getAddressesFromExtension({
       state: {
-        externals: { getAccounts }
-      }
+        externals: { getAccountsFromExtension }
+      },
+      commit
     }) {
-      getAccounts() // response will be handled by extension message listener
+      const wallets = await getAccountsFromExtension()
```

Comment:
This is now not working anymore I think

Reply:
Woops, it should still set the accounts, but it's not necessary code here.